client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,10 +45,44 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
+				<p>
+					<FormLabel>
+						<FormRadio
+							name="auth_method"
+							value="app-based"
+							defaultChecked={ true }
+							onChange={ setAuth }
```

Comment:
Note that once we move `setAuth` to be a method of the component class, this will have to be changed to `this.setAuth`.